app/helpers/api_forms_helper.rb

```diff
@@ -70,5 +70,19 @@ def map_form_field_type(data_type)
       'String'
     end
   end
+
+  # custom_message format: api_resource.properties["first_name"]
+  def parse_snippet(custom_snippet, api_resource)
+    return unless custom_snippet.present?
+
+    parsed_snippet = custom_snippet
+    # couldn't gsub directly because of escape characters added by ruby
+    # eval failed on "\#{api_resource.properties[\"String\"]}"
+    custom_snippet.scan(/\#\{(.*?)\}/).each do |code|
+      # TODO: sanitize code before eval  
```

Comment:
@Pralish After we merge https://github.com/restarone/violet_rails/pull/625 can we hook up the concern to this ?